### PR TITLE
China plugin tweaks and map fragment addition

### LINKED ISSUE #725: Add docs about China Plugin's ChinaSupportMapFragment
Fragment support was recently added to the Mapbox China Plugin for Android via the `ChinaSupportMapFragment` class. I/we need to add docs to https://www.mapbox.com/android-docs/plugins/overview/china about this new class and how to use it. 

cc @Chaoba

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #725 by adding mention of the `ChinaSupportMapFragment`, which was recently added to the Mapbox China Plugin for Android. This pr also makes other housekeeping tweaks to the plugin's page.

cc fyi @springmeyer @Chaoba

### CODE REVIEW

File: src/pages/plugins/overview/china.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ prependJs:
...
+Traditional map services are either blocked in China or suffer from slow internet connections. Our mapbox.cn infrastructure allows for unparalleled speed advantages for anyone using our maps in China or through Chinese mobile carriers internationally. This Mapbox China Plugin for Android specifically bundles the Mapbox Maps SDK for Android and automatically configures the Maps SDK to ensure that the correct endpoints are being called. Accurate endpoints ensure that a mobile device retrieves the correct map tiles, map styles, and other location information. Additionally, the plugin allows for annotation shifting, which ensures all your overlays and annotations are accurately depicted on the map.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: >This Mapbox China Plugin for Android

to 

>The Mapbox China Plugin for Android

?

File: src/pages/plugins/overview/china.md [hunk 2]
```diff
@@ -48,17 +48,18 @@ Another option is to drop the architectures you do not need to support for your
...
+The plugin has wrapper classes to be used in place of their "typical" counterparts. For example, in the regular Maps SDK for Android, you'd use the `MapView` inside your activities' layouts. When using this plugin, a lint error will appear telling you to use the `ChinaMapView` instead, which wraps the `MapView` object. Under the hood, the `ChinaMapView` class does nothing more but set default values for optimal performance inside China. This includes requesting map tiles from our Chinese servers. In the provided chart below, you'll find a list of the objects which you'd typically use in the Maps SDK for Android and their counterparts found inside of this plugin.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: >Under the hood

I wonder if we should avoid this phrase in an effort to `Avoid colloquialisms or idioms` per the style guide.

File: src/pages/plugins/overview/china.md (hunk 1 above)
Comment (langsmith) [reply src/pages/plugins/overview/china.md]: Fixed

File: src/pages/plugins/overview/china.md (hunk 2 above)
Comment (langsmith) [reply src/pages/plugins/overview/china.md]: Yep. Got rid of it. Sentence now reads as

>The `ChinaMapView` class does nothing more but set default values for optimal performance inside China

**Pull request merged**